### PR TITLE
Travis schedule change

### PR DESCRIPTION
**Pull request by tatsta**

### CODE REVIEW

**Comment by tomazpu** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ src/Response/Response.php  97
+ src/Transaction/PaysafecardTransaction.php  100
+ src/Entity/RiskInfo.php  76
+ src/Entity/Browser.php  35
+ src/Entity/CustomField.php  85
+ src/Entity/Status.php  100
+ src/Transaction/PayPalTransaction.php  100
+ src/Entity/Card.php  100
+ src/Response/SuccessResponse.php  100
+ src/Transaction/BancontactTransaction.php  100
+ src/Entity/Mappable.php  6
+ src/Response/FailureResponse.php  100
+ src/Entity/FormFieldMap.php  100
+ src/Transaction/CreditCardTransaction.php  99
+ src/Response/FormInteractionResponse.php  100
+ src/Entity/Basket.php  92
+ src/Config/PaymentMethodConfig.php  100
+ src/Entity/CompanyInfo.php  100
+ src/Transaction/UpiTransaction.php  100
+ src/Response/InteractionResponse.php  100
+ src/Transaction/EpsTransaction.php  40
+ src/Config/CreditCardConfig.php  100
+ src/Entity/SubMerchantInfo.php  100
+ src/Entity/Periodic.php  100
+ src/Entity/StatusCollection.php  100
+ src/Transaction/SepaCreditTransferTransaction.php  100
+ src/Transaction/AlipayCrossborderTransaction.php  100
+ src/Transaction/SepaDirectDebitTransaction.php  100
+ src/Transaction/SofortTransaction.php  100
+ src/Entity/Address.php  100
+ src/Mapper/RequestMapper.php  90
+ src/Transaction/MasterpassTransaction.php  100
+ src/Transaction/CustomFieldTransaction.php  85
+ src/Entity/PaymentDetails.php  90
+ src/Entity/AccountInfo.php  73
+ src/Transaction/PtwentyfourTransaction.php  100
+ src/Transaction/PayolutionBtwobTransaction.php  100
+ src/Transaction/WeChatTransaction.php  100
+ src/Entity/TransactionDetails.php  98
+ src/Config/MaestroConfig.php  100
+ src/Config/SepaConfig.php  100
+ src/Entity/Device.php  91
+ src/Transaction/Risk.php  100
+ src/BackendService.php  43
+ src/Config/Config.php  98
+ src/Transaction/Transaction.php  90
+ src/Transaction/RatepayDirectDebitTransaction.php  100
+ src/Entity/Item.php  100
+ src/Transaction/UpopTransaction.php  100
+ src/TransactionService.php  53
+ src/Transaction/SepaBtwobTransaction.php  100
+ src/Entity/Amount.php  100
+ src/Transaction/RatepayInvoiceTransaction.php  100
+ src/Transaction/IdealTransaction.php  100
+ src/Transaction/RatepayInstallmentTransaction.php  100
+ src/Mapper/ResponseMapper.php  100
+ src/Entity/AccountHolder.php  85
+ src/Entity/BankAccount.php  100
+ src/Transaction/RatepayTransaction.php  100
+ src/Entity/Redirect.php  39
+ src/Entity/Mandate.php  100
+ src/Transaction/GiropayTransaction.php  100
+ src/Transaction/PoiPiaTransaction.php  100
+ src/Transaction/PayolutionInvoiceTransaction.php  100
+ src/Transaction/MaestroTransaction.php  100
+ src/Entity/CustomFieldCollection.php  98
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Wirecard/paymentSDK-php/pullRequest?prid=4489825&bid=15144097)

File: .travis.yml [hunk 1]
```diff
@@ -1,71 +1,87 @@
...
-    - CHECK_CS=true
-    - CREATE_DOCS=true
+      - CHECK_CS=true
+      - CREATE_DOCS=true
   - php: 7.1
-    if: type != cron
+    if: type != cron AND commit_message != env(MANUAL_UITEST_TRIGGER_COMMIT)
     env:
-    - MUTATION_TESTS=true
+      - MUTATION_TESTS=true
```
Comment (tomazpu) [inline .travis.yml]: I suggest disabling the mutation tests, till we define the guides for it.

File: tests/_support/AcceptanceTester.php [hunk 2]
```diff
@@ -225,6 +225,7 @@ public function iClick($object)
     {
         $this->currentPage->prepareClick();
         $this->waitForElementVisible($this->getPageElement($object), 30);
```
Comment (tomazpu) [inline tests/_support/AcceptanceTester.php]: Is not waitForElementClickable() only enough as an element is not clickable before it is visible?
I also saw that this class has no namespace, is there any reason for this?

File: .travis.yml (hunk 1 above)
Comment (tatsta) [reply .travis.yml]: I have a task in the backlog for changing schedule on mutation tests, but this will be done after this PR

**Pull request merged**